ql/src/test/queries/clientpositive/multi_insert_gby5.q

```diff
@@ -0,0 +1,10 @@
+set hive.cbo.fallback.strategy=NEVER;
+
+create table tbl1 (key int, f1 int);
+create table tbl2 (f1 int) partitioned by (key int);
+
+FROM (select key, f1 FROM tbl1 where key=5) a
```

Comment:
The query doesn't fail but we don't check the EXPLAIN and we don't check the results either so we can't be sure that it is working fine. Let's include and verify the EXPLAIN to be on the safe side.

Reply:
Added explain plan